<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WControlBola</class>
 <widget class="QWidget" name="WControlBola">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bParar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bColor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QFrame" name="fBotonesChoques">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
